webviz_subsurface/plugins/_simulation_time_series/views/subplot_view/_subplot_view.py

```diff
@@ -0,0 +1,900 @@
+                        ResamplingFrequencySettings.Ids.RESAMPLING_FREQUENCY_DROPDOWN,
+                    ),
+                    "value",
+                ),
+                Input(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.FILTER_REALIZATION_SETTINGS,
+                        FilterRealizationSettings.Ids.REALIZATIONS_FILTER_SELECTOR,
+                    ),
+                    "value",
+                ),
+                Input(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.FILTER_REALIZATION_SETTINGS,
+                        FilterRealizationSettings.Ids.STATISTICS_FROM_RADIO_ITEMS,
+                    ),
+                    "value",
+                ),
+                Input(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.RESAMPLING_FREQUENCY_SETTINGS,
+                        ResamplingFrequencySettings.Ids.RELATIVE_DATE_DROPDOWN,
+                    ),
+                    "value",
+                ),
+                Input(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.TIME_SERIES_SETTINGS,
+                        TimeSeriesSettings.Ids.GRAPH_DATA_HAS_CHANGED_TRIGGER,
+                    ),
+                    "data",
+                ),
+            ],
+            [
+                State(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.ENSEMBLE_SETTINGS,
+                        EnsemblesSettings.Ids.CREATED_DELTA_ENSEMBLES_STORE,
+                    ),
+                    "data",
+                ),
+                State(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.TIME_SERIES_SETTINGS,
+                        TimeSeriesSettings.Ids.VECTOR_CALCULATOR_EXPRESSIONS,
+                    ),
+                    "data",
+                ),
+                State(
+                    self.settings_group_unique_id(
+                        SubplotView.Ids.ENSEMBLE_SETTINGS,
+                        EnsemblesSettings.Ids.ENSEMBLES_DROPDOWN,
+                    ),
+                    "options",
+                ),
+            ],
+        )
+
+        # pylint: disable=too-many-arguments, too-many-locals, too-many-branches, too-many-statements
+        def _update_graph(
+            vectors: List[str],
+            selected_ensembles: List[str],
+            visualization_value: str,
+            statistics_option_values: List[str],
+            fanchart_option_values: List[str],
+            trace_option_values: List[str],
+            subplot_owner_options_value: str,
+            resampling_frequency_value: str,
+            selected_realizations: List[int],
+            statistics_calculated_from_value: str,
+            relative_date_value: str,
+            __graph_data_has_changed_trigger: int,
```

Comment:
Why is this argument super private?

Reply:
Argument denoted as it works as a callback trigger only - value not utilized. Denoted with one `_` prefix too much. Corrected to `_graph_data_has_changed_trigger`